ios/FluentUI.Demo/FluentUI.Demo/Demos/BottomSheetDemoController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import Foundation
+import FluentUI
+
+class BottomSheetDemoController: DemoController {
+// MARK: Lifecycle methods
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setupMainPersonaListView()
+        setupBottomSheet()
+    }
```

Comment:
Keeping this one because the superclass adds a scrollview in viewDidLoad which ends up over all of our content and breaks stuff. This demo controller will change in the future to be more comprehensive and will make proper use of the superclass container views.